00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,593 @@
+  makes it an immediate priority. If it's not considered urgent, it will be
+  dropped in the project's backlog for later consideration.
+
+- PRs represent work that has already been done and needs to be reviewed before
+  being merged into the main codebase. As such, they will almost always be
+  attached to an issue in the current sprint, and should be treated as an
+  extension of that issue. The sole purpose of triaging a new PR is to ensure it
+  has reviewers requested and assigned. Once that's done, it gets put in the
+  current sprint.
+
+### Project Board
+
+A common concept in many project workflows, the **project board** is also how the
+Enarx project organizes its work.
+
+Currently, the project uses multiple area-specific boards. This proposal
+abandons that idea and instead uses a single, unified project board that
+contains the entire project backlog and all current work.
+
+The board will be organized with the following columns:
+
+- backlog: collection of issues not currently being worked on
+- nominated: issues nominated by team members for the next sprint (planning)
+- accepted: issues the team should work on next sprint (planning)
+- assigned: issues actively being worked on by an assigned team member
+- under review: all submitted PRs go here while awaiting review
+- done: all issues and PRs go here when done or merged, respectively
+
+Here's a quick diagram:
+
+![Complete issue workflow diagram](./project-board.svg)
+
+In more technical terms, this would be an organization-level Github project
+board. This will allow for issues and PRs across _any_ Enarx repository (not
+just the main one) to be included, which supports modularization of the project
+as it grows.
+
+## Meetings
+
+The concepts and processes introduced above are put into action in the form of
+several team meetings that happen throughout the process of a single sprint.
+
+To start, consider this high-level overview of how an issue progresses through
+the process, beginning with its opening and continuing all the way until it's
+considered done:
+
+![Complete issue workflow diagram](./full-process.svg)
+
+This flowchart introduces the three crucial events of the process, each of
+which is encapsulated by its own team meeting:
+ 
+ - **Planning**, a meeting representing the start of a new sprint
+ - **Daily Standup**, the team's daily meeting where, among other things,
+   **Triage** is performed.
+ - **Review**, a meeting representing the end of a sprint where, among other
+   things, **Retrospective** is performed.
+
+Once an issue is filed, it will be categorized and discussed in Triage. If
+urgent, it's injected straight into the current sprint; if not, it gets pushed
+to the backlog. From there, the rest of the cycle will carry it through to
+completion: Planning will sooner or later land it as part of a sprint, where it
+will become a team member's action item to complete. At the end of the sprint,
```

Comment:
> Planning will sooner or later land it as part of a sprint

Probably, but there is no guarantee of this: what happens if an issue keeps staying in the backlog?

Not needed right now, but we should probably eventually add a time to cull old issues that are not getting put in sprints, as they have become stale. If they are in fact not stale, why they are not getting added to sprints should in itself be a question worth answering, as it's a sign of some other problem.

Reply:
Good point. I will add something to the Future Possibilities section.